sentry-sidekiq/lib/sentry/sidekiq/sentry_context_middleware.rb

```diff
@@ -26,8 +26,19 @@ def call(_worker, job, queue)
         scope.set_span(transaction) if transaction
 
         begin
-          yield
-        rescue
+          Sentry.with_child_span(op: "queue.process", description: "Process #{worker.class.name}") do |span|
+            # Set span data
+            if span
+              span.set_data("messaging.message.id", job["jid"])
+              span.set_data("messaging.destination.name", queue)
+              span.set_data("messaging.message.body.size", job.to_json.size)
+              span.set_data("messaging.message.receive.latency", ((Time.now.to_f - job["created_at"]) * 1000).to_i)
```

Comment:
You should use enqueued_at instead. Using created_at will cause incorrect calculations for scheduled jobs.